contrib/sarplus/python/setup.py

```diff
@@ -39,28 +37,23 @@ def __str__(self):
     classifiers=[
         "Development Status :: 5 - Production/Stable",
         "License :: OSI Approved :: MIT License",
-        "Programming Language :: Python :: 3.6",
-        "Programming Language :: Python :: 3.7",
-        "Programming Language :: Python :: 3.8",
-        "Programming Language :: Python :: 3.9",
-        "Programming Language :: Python :: 3.10",
+        "Programming Language :: Python :: 3",

```

Comment:
We could consider keeping the specific python versions, if we only put python 3, people can think that 3.5 can be used, or future versions (3.11) that we have not test. What do you think?

One question that is not clear, python 3.6 is included or not? FYI @anargyri

Reply:
3.6 is currently included. What is the reason for excluding it from SARplus?

Reply:
@anargyri The latest [setuptools](https://pypi.org/project/setuptools/) now only supports Python 3.7+, and I also just found Python 3.6 has reached end-of-life.  Please see
* [Release Python 3.6](https://www.python.org/downloads/release/python-360/)
* [Status of Python branches](https://devguide.python.org/#status-of-python-branches)
But I reverted to Python 3.6 in alignment with PySpark now.

@miguelgfierro Agree.

Reply:
Good point about 3.6. But some people may still be using it for whatever reason, so we could still allow it.